index.html

```diff
@@ -3,177 +3,57 @@
+    <div class="confetti-wrapper display-none"></div>
     <!--MAIN CONTENT-->
     <div class="container">
-        <div style="display: flex; align-items: center">
-            <h2 class="head-1" style="font-size: 6.5vh; color: white;">Dear Crush</h2>
+        <div class="container-header">
+            <h2 class="color-white">Dear Crush</h2>
             <img width="150" height="150" src="https://media.tenor.com/cofRHcGGOfoAAAAi/shy-cute.gif" alt="cute"/>
         </div>
-        <div style="display: flex; flex-direction: column; justify-content: center;align-items: center; ">
-            <h1 style="font-size: 7.5vh;text-align: center; color: white; margin-bottom: 24px;">Will you go out with me?</h1>
-            <img class="please-cat" src="https://media.tenor.com/_W2FjRBFfkcAAAAC/cat-please.gif" alt="cute cat image" style="height: 32vh;border-radius: 8px;"/>
+        <div class="container-content">
+            <h1 class="color-white">Will you go out with me?</h1>
+            <img class="please-cat" src="https://media.tenor.com/_W2FjRBFfkcAAAAC/cat-please.gif" alt="cute cat image"/>
         </div>
-        <div style="margin: 24px 197px 64px 32px;">
-            <button id="showBtn" class="yes-btn" onclick="showConfetti()" style="margin-right: 24px;">Yes</button>
+        <div class="action-button">
+            <button id="showBtn" class="yes-btn" onclick="showConfetti()">
+                <span class="heart"></span>
+                <span class="heart-outer"></span>
+                <span class="yes-btn-text">Yes</span>
+            </button>
             <button class="no-btn" onmouseover="crazyButton()" id="btn">No</button>
         </div>
-        <div id="celebrate" style="display: none">
-            <img style="position: absolute;left: 0; top:15%; z-index: -2" src="https://media.tenor.com/qg8K8VOmzJwAAAAi/party-popper-confetti.gif" alt="celebrate"/>
-            <img style="position: absolute;right: 0; top:15%; transform: scaleX(-1); z-index: -2" src="https://media.tenor.com/qg8K8VOmzJwAAAAi/party-popper-confetti.gif" alt="celebrate"/>
-            <img style="position: absolute;right: 0; bottom:0;z-index: -2" src="https://media.tenor.com/DXL84KB6lscAAAAi/confetti-cute.gif" alt="celebrate"/>
-            <img style="position: absolute;left: 0; bottom:0;z-index: -2; transform: scaleX(-1);" src="https://media.tenor.com/DXL84KB6lscAAAAi/confetti-cute.gif" alt="celebrate"/>
+        <div class="display-none celebrate">
+            <img src="https://media.tenor.com/qg8K8VOmzJwAAAAi/party-popper-confetti.gif" alt="celebrate"/>
+            <img src="https://media.tenor.com/DXL84KB6lscAAAAi/confetti-cute.gif" alt="celebrate"/>
         </div>
-        <b style="color:white;font-family: sans-serif">-Sincerely, Eshan</b>
+       <h3 class="color-white">-Future Boyfriend, Eshan</h3>
     </div>
     <!--DIALOG CONTENT-->
     <dialog id="showDialog">
-        <div style="border: 5px double #DA291C;border-radius: 4px; padding:0 24px 24px">
```

Comment:
glassmorphism effect on the modal is not looking good, I think there is a better way, can't think of now but will definitely change in future